stac_mjx/main.py

```diff
@@ -63,36 +69,53 @@ def run_stac(
 
     stac = Stac(xml_path, cfg, kp_names)
 
+    compute_velocity_fn = partial(
+        utils.compute_velocity_from_kinematics,
+        dt=stac._mj_model.opt.timestep,
+        freejoint=stac._freejoint,
+    )
+    # Initialize function to infer velocity from kinematics
+    vmap_compute_velocity_fn = jax.vmap(compute_velocity_fn)
+
     # Run fit_offsets if not skipping
     if cfg.stac.skip_fit_offsets != 1:
-        fit_data = kp_data[: cfg.stac.n_fit_frames]
-        logging.info(f"Running fit. Mocap data shape: {fit_data.shape}")
-        fit_data = stac.fit_offsets(fit_data)
-
-        logging.info(f"saving data to {fit_offsets_path}")
-        io.save(fit_data, fit_offsets_path)
+        kps = kp_data[: cfg.stac.n_fit_frames]
+        print(f"Running fit. Mocap data shape: {kps.shape}")
+        fit_offsets_data = stac.fit_offsets(kps)
+        print(f"saving data to {fit_offsets_path}")
+        io.save_data_to_h5(
+            config=cfg, file_path=fit_offsets_path, **fit_offsets_data.as_dict()
+        )
+        (fit_offsets_data, fit_offsets_path)
 
     # Stop here if not doing ik only phase
     if cfg.stac.skip_ik_only == 1:
-        logging.info("skipping ik_only()")
+        print("skipping ik_only()")
```

Comment:
Let's give the users a bit more of an informative error message? Maybe:

```suggestion
        print("Skipping IK-only phase. Only offsets will be returned. To change this behavior, set cfg.stac.skip_ik_only to 0.")
```